src/app/ui-components/sidebar-right/sidebar-right.component.html

```diff
@@ -1,32 +1,123 @@
-<div class="col-12 sidebar-right-wrapper p-0">
-    <div class="row p-0 m-0 mb-1">
-        <div
-            class="col-4 p-0"
-            *ngFor="let element of headerElements">
-            <button
-                class="header-button"
-                (click)="element.clickCallback()"
-                id="label_{{element.label}}">
-                <img
-                    class="mt-3"
-                    [src]="element.icon"
-                    alt="" />
-                <p>{{element.label}}</p>
-            </button>
-        </div>
-    </div>
+<div class="col sidebar-right-wrapper h-100">
     <div class="row">
-        <div
-            class="sidebar-element row"
-            *ngFor="let sidebarElement of sidebarElements"
-            routerLinkActive="sidebar-element-active">
-            <div class="col-auto">
-                <a [routerLink]="sidebarElement.getUUID()"
-                    ><img
-                        *ngIf="elementIcon"
-                        [src]="elementIcon" />
-                    {{ sidebarElement.getName() }}</a
-                >
+        <div class="mt-4">
+            <div
+                class="row"
+                *ngIf="lStorage == 'chat'">
+                <label class="text-center">Voice interaction</label>
+                <div class="col">
+                    <div class="row mt-1 mb-3">
+                        <div
+                            class="col-11 ml-3 onOff"
+                            [ngClass]="vaState == true ? 'on' : 'off'">
+                            <div class="row">
+                                <button
+                                    type="button"
+                                    class="col btn text-truncate element text-center p-3"
+                                    id="toggle-voice-assistant"
+                                    (click)="toggleVoiceAssistant()"
+                                    alt="Toggle for turning the Voice Assistant on and off"
+                                    value="{{vaState}}">
+                                    <div
+                                        *ngIf="vaState == true; then thenBlock else elseBlock"></div>
+                                    <ng-template #thenBlock
+                                        ><label class="mb-0"
+                                            >ON</label
+                                        ></ng-template
+                                    >
+                                    <ng-template #elseBlock
+                                        ><label class="mb-0"
+                                            >OFF</label
+                                        ></ng-template
+                                    >
+                                </button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
```

Comment:
sidebar should not contain any voice-assistant-specific content. This could f.e. be replaced with `<ng-content></ng-content>` and then the content can be moved between the `<app-sidebar-right></app-sidebar-right>` tags in the voice-assistant-chat component. The voice-assistant related functionality can then also be moved accordingly